tencentcloud/tke/v20180525/models.go

```diff
@@ -720,8 +720,8 @@ type ClusterBasicSettings struct {
 	// 容器的镜像版本，"DOCKER_CUSTOMIZE"(容器定制版),"GENERAL"(普通版本，默认值)
 	OsCustomizeType *string `json:"OsCustomizeType,omitempty" name:"OsCustomizeType"`
 
-	// 是否开启节点的默认安全组(默认: 否，Aphla特性)
-	NeedWorkSecurityGroup *bool `json:"NeedWorkSecurityGroup,omitempty" name:"NeedWorkSecurityGroup"`
+	// 是否开启节点的默认安全组(默认: 否，Alpha特性)
+	NetWorkSecurityGroup *bool `json:"NetWorkSecurityGroup,omitempty" name:"NetWorkSecurityGroup"`
```

Comment:
It actually should be 'Need' instead of 'Net', it means if you want to enable this feature .
This file is auto generated, it should not be merged.
I will ask the responsible team to correct the typo of "alpha"